### PR TITLE
[13.0][MIG] stock_quantity_history_location

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Inventory Report has been modified slightly, this firsts commits are a standard migration of features. 
It can be reviewed, ideas are welcomed for improving usability.

CC @ForgeFlow

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

File: stock_quantity_history_location/__manifest__.py [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Stock Quantity History Location",
     "summary": "Provides stock quantity by location on past date",
-    "version": "12.0.1.0.1",
+    "version": "13.0.1.0.1",
     "license": "AGPL-3",
-    "author": "Eficent," "Odoo Community Association (OCA)",
+    "author": "ForgeFlow S.L.," "Odoo Community Association (OCA)",
```
Comment (LoisRForgeFlow) [inline stock_quantity_history_location/__manifest__.py]: Just ForgeFlow

Comment (HviorForgeFlow) [reply stock_quantity_history_location/__manifest__.py]: Done!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Pull request merged**